spec/sierra_bib_spec.rb

```diff
@@ -149,7 +149,7 @@
     end
   end
 
-  describe 'find_oclcnum' do
+  describe 'oclcnum' do

```

Comment:
Testing initializing a new SierraBib 'b' and calling b.oclcnum without explicitly doing any 'get_oclcnum'
Tests pass.